pythnet/pythnet_sdk/src/tests/test_message.rs

```diff
@@ -0,0 +1,96 @@
+use {
```

Comment:
I know it was here on the client as well. But I think generally unit tests go within the same file as the definition and integration tests go to `tests` directory.